test/golden/routes_golden_test.dart

```diff
@@ -152,6 +152,20 @@ void main() {
         await tester.pumpAndSettle();
       }, goldenCaptureCallback: () => tester.screenMatchesGolden(tester, 'selection_route.selection_route'));
     });
+
+    testAppGoldens('selection_route_settings', (WidgetTester tester) async {
+      await tester.runAppTest(() async {
+        HomeRouter.instance.goto(HomeRoutes.factory.content<Playlist>(playlistWith()));
+        await tester.pumpAndSettle();
+        await tester.tap(find.byIcon(Icons.add_rounded).first);
+        await tester.pumpAndSettle();
+        await tester.tap(find.descendant(
+            of: find.byType(SelectionRoute),
+            matching: find.byIcon(Icons.settings_rounded),
+        ));
+        await tester.pumpAndSettle();
+      }, goldenCaptureCallback: () => tester.screenMatchesGolden(tester, 'selection_route.selection_settings'));
```

Comment:
please make the test name and `selection_route.<what_goes_here>` to match with each other, i.e. the test should be named `selection_settings`

Reply:
The naming of the golden files is `group.name`, this test would be the only one to not follow this rule.